### DISCUSSION TITLE
Suggested change API

### DISCUSSION THREAD

**Discussion by jirfag** [community/community#24848] [Apps, API and Webhooks]
<p>Hi! Is there documentation about suggested changes API (beta)?</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/jirfag">@jirfag</a>,</p>
<p>Thanks for being here! Here is a link to <a href="https://blog.github.com/changelog/2018-10-16-suggested-changes/" rel="nofollow noopener">the blog post</a> and other docs.</p>
<p>I hope this helps!</p>
<p>Andrea</p>

**Comment by jirfag** [comment]
<p>Thank you!</p>
<p>But there is no information in this blog post or doc how to use it by API (not in web interface)</p>

**Comment by jirfag** [comment]
<p>I got it, sorry for misunderstanding</p>

**Comment by romeara** [comment]
<p>I am looking at the linked docs, and I’m having trouble finding my way to information on doing this programmatically via API - <a class="mention" href="/jirfag">@jirfag</a> , where were you looking in the doc when you said you got it? All of it seems to be related to doing reviews through the UI</p>
<p>EDIT:</p>
<p>Think I figured it out - there is a specific set of formatting that can be used in line comments (it appears). For others finding this thread, the <a>Commenting on a Pull Request</a> document has a snippet like this:</p>
<pre><code>```suggestion
else if 1 &gt; x
```
</code></pre>

**Comment by nalbion** [comment]
Can you make comments on files that are not changed (yet) by the PR?
